config/stack/manifests/app.yaml

```diff
@@ -0,0 +1,51 @@
+# Human readable title of application.
```

Comment:
Can we use Helm chart as the general packaging approach and leverage helm hub/OCI registry for this process?

I'm asking since this file looks quite overlap with `Chart.yaml`.

Reply:
This addon requires crossplane to be installed since it makes use of both the `ApplicationConfiguration` controller and the `KubernetesApplication` controller, so there doesn't seem to be much of a use-case here for a separate chart. I could see a potential benefit of allowing the `local` implementation to be installed in a stand-alone fashion, but this one is essentially unusable without crossplane handling the scheduling to remote clusters.